app/server.go

```diff
@@ -1781,6 +1781,51 @@ func (s *Server) startMetricsServer() {
 	s.Log.Info("Metrics and profiling server is started", mlog.String("address", l.Addr().String()))
 }
 
+func (s *Server) sendLicenseUpForRenewalEmail(users map[string]*model.User, license *model.License) *model.AppError {
+	key := model.LICENSE_UP_FOR_RENEWAL_EMAIL_SENT + license.Id
+	if _, err := s.Store.System().GetByName(key); err == nil {
+		return nil
+	}
```

Comment:
Should we handle the error case here? 

Also, can we make a comment on what is happening here? We are returning early because the email was already sent

Reply:
I think for this we want the case where the err is nil because it confirms that we already did this action and we therefore don't want to proceed any further. The opposite where the err is not nil we want to continue because the key cannot be retrieved.

Add a comment to the code as suggested.

Reply:
Thanks for the explanation here. Sounds good 👍